lcUI/cadmdichild.cpp

```diff
@@ -385,4 +385,15 @@ void CadMdiChild::cancelCurrentOperations() {
 
 lc::UndoManager_SPtr CadMdiChild::undoManager() const {
     return _undoManager;
+}
+
+std::shared_ptr<LCViewer::Cursor> CadMdiChild::cursor() const {
+	return _cursor;
+}
+
+unsigned int CadMdiChild::id() {
```

Comment:
should we change the _id type to "unsigned long long"?

even though it's mostly 32 bit for "unsigned int", the C++ standard only requires "unsigned int" type to be at least 16 bit.

I suppose we need 64 bit to avoid collision on a busy system.